stacker/tests/test_config.py

```diff
@@ -398,6 +400,36 @@ def test_render_parse_load_namespace_fallback(self):
         config.validate()
         self.assertEquals(config.namespace, "prod")
 
+    def test_raise_constructor_error_on_duplicate_key(self):
+        yaml_config = """
+        namespace: prod
+        stacks:
+          - name: vpc
+            class_path: blueprints.VPC
+            class_path: blueprints.Fake
+        """
+        with self.assertRaises(ConstructorError):
+            parse(yaml_config)
+
+    def test_raise_error_on_duplicate_stack_name(self):
+        yaml_config = """
+        namespace: prod
+        stacks:
+          - name: vpc
+            class_path: blueprints.VPC
+          - name: vpc
+            class_path: blueprints.VPC
```

Comment:
Can you change this class_path to something different?  Just want to make sure it's only testing the stack name, and nothing else.

Reply:
Also, can you write a test that tests that the dictionary version of stacks fails the unique validation?

Reply:
fixed